<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NFTAddDialog</class>
 <widget class="QWidget" name="NFTAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1245</width>
    <height>809</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <item>
    <widget class="QFrame" name="frame2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="2">
         <widget class="QLabel" name="lblAction">
          <property name="text">
           <string>Add New NFT - v1.2   &lt;br&gt;</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_213">
          <property name="toolTip">
           <string>Enter NFT Name</string>
          </property>
          <property name="text">
           <string>NFT Name:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtReserved</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="txtName">
          <property name="toolTip">
           <string>Address</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>150</height>
           </size>
          </property>
          <property name="toolTip">
           <string>NFT Description</string>
          </property>
          <property name="text">
           <string>Description:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtDescription</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPlainTextEdit" name="txtDescription">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>140</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_912">
          <property name="toolTip">
           <string>Reserve Amount</string>
          </property>
          <property name="text">
           <string>Buy-it-Now Amount:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtBuyItNowAmount</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLineEdit" name="txtBuyItNowAmount">
          <property name="toolTip">
           <string>Buy It Now Amount (The amount you will accept for an immediate sale)</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLineEdit" name="txtReserveAmount">
          <property name="toolTip">
           <string>Reserve Amount</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_512">
          <property name="toolTip">
           <string>Reserve Amount</string>
          </property>
          <property name="text">
           <string>Reserve Amount:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtReserveBidAmount</cstring>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QLineEdit" name="txtMinimumBidAmount">
          <property name="toolTip">
           <string>Enter Reserve Amount (The auction price you will accept that triggers a sale)</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly</set>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_212">
          <property name="toolTip">
           <string>The lowest bid you will consider.</string>
          </property>
          <property name="text">
           <string>Minimum Bid Amount:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtMinimumBidAmount</cstring>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_22">
          <property name="toolTip">
           <string>Lo-Quality URL</string>
          </property>
          <property name="text">
           <string>Lo-Quality URL:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtLoQualityURL</cstring>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QLineEdit" name="txtLoQualityURL">
          <property name="toolTip">
           <string>LoQuality URL</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_72">
          <property name="toolTip">
           <string>Hi-Quality URL</string>
          </property>
          <property name="text">
           <string>Hi-Quality URL:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtHiQualityURL</cstring>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QLineEdit" name="txtHiQualityURL">
          <property name="toolTip">
           <string>HiQuality URL</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label194">
          <property name="toolTip">
           <string>Owner Address</string>
          </property>
          <property name="text">
           <string>(CPK) Owner Address:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="QLineEdit" name="txtOwnerAddress">
          <property name="toolTip">
           <string>Address</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="12" column="2">
         <widget class="QComboBox" name="cmbNFTType"/>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_31">
          <property name="toolTip">
           <string>NFT Type</string>
          </property>
          <property name="text">
           <string>NFT Type:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="14" column="2">
         <widget class="QCheckBox" name="chkMarketable">
          <property name="maximumSize">
           <size>
            <width>777</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Show in web Marketplace? (Item is for sale, or orphan is sponsorable)?</string>
          </property>
         </widget>
        </item>
        <item row="15" column="2">
         <widget class="QCheckBox" name="chkDeleted">
          <property name="maximumSize">
           <size>
            <width>777</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Delete this NFT?</string>
          </property>
         </widget>
        </item>
        <item row="16" column="7">
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </item>
        <item row="16" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="16" column="2">
         <widget class="QPushButton" name="btnSubmit">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;Submit NFT</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-style:italic;&quot;&gt;BiblePay Non-Fungible Tokens&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="txtInfo">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtDescription</tabstop>
  <tabstop>txtMinimumBidAmount</tabstop>
  <tabstop>txtOwnerAddress</tabstop>
  <tabstop>cmbNFTType</tabstop>
  <tabstop>chkMarketable</tabstop>
  <tabstop>btnSubmit</tabstop>
 </tabstops>
 <resources>
  <include location="../biblepay.qrc"/>
 </resources>
 <connections/>
</ui>
